native/explorer/src/series.rs

```diff
@@ -259,18 +259,6 @@ pub fn s_slice(series: ExSeries, offset: i64, length: usize) -> Result<ExSeries,
     Ok(ExSeries::new(series.slice(offset, length)))
 }
 
-#[rustler::nif(schedule = "DirtyCpu")]
```

Comment:
shifted to elixir side for eager series. Need to change this to align with `ignore_nulls`.